modules/setting/indexer.go

```diff
@@ -53,6 +56,16 @@ func loadIndexerFrom(rootCfg ConfigProvider) {
 		Indexer.IssuePath = filepath.ToSlash(filepath.Join(AppWorkPath, Indexer.IssuePath))
 	}
 	Indexer.IssueConnStr = sec.Key("ISSUE_INDEXER_CONN_STR").MustString(Indexer.IssueConnStr)
+
+	if Indexer.IssueType == "meilisearch" {
+		u, err := url.Parse(Indexer.IssueConnStr)
+		if err != nil {
+			log.Warn("Failed to parse ISSUE_INDEXER_CONN_STR: %v", err)
+			u = &url.URL{}
+		}
+		Indexer.IssueConnAuth, _ = u.User.Password()
```

Comment:
Then Indexer.IssueConnStr should be assign again.

Reply:
Does it? It's extra computation and string generation, when the meilisearch SDK would still behave the same.

Reply:
How about
```suggestion
                Indexer.IssueConnAuth, _ = u.User.Password()
                u.User = nil
                Indexer.IssueConnStr = u.String()
```